livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -1167,4 +1158,43 @@ void Lddc::checkTimeSync(diagnostic_updater::DiagnosticStatusWrapper & stat)
     stat.summary(whole_level, time_sync_dict_.at(whole_level));
   }
 }
+
+void Lddc::checkConnect(diagnostic_updater::DiagnosticStatusWrapper & stat)
+{
+  if (lidar_count_ == 0) {
+    stat.summary(DiagStatus::WARN, "No LiDARs Connected");
+    return;
+  }
+
+  int whole_level = DiagStatus::OK;
+  std::string error_str = "";
+
+  for (const auto & lidar : lds_->connected_lidars_) {
+    int level = DiagStatus::OK;
+
+    const auto & broadcast_code = lidar.first;
+    const auto & device_info = lidar.second;
+
+    if (device_info == nullptr) {
+      error_str = "LiDAR disconnected";
+      stat.add(broadcast_code, "disconnected");
+      continue;
+    }
+
+    if (device_info->state == kLidarStateInit) {
+      stat.addf(broadcast_code, "%d%%", device_info->status.progress);
```

Comment:
I feel `progress: %d[%%]` is better.

Reply:
This line is used other check function. So I feel changing it in other PR is better.

Reply:
Yes, it's okay!

Reply:
Will we open a new issue?

Reply:
I meant there are many other similar lines like https://github.com/tier4/livox_ros2_driver/blob/dd005558fa82fc370a2e7f58106cd967640e35f4/livox_ros2_driver/livox_ros2_driver/lddc.cpp#L738
So I just send PR of changing all ot them. No need to opening an issue.

Reply:
@kosuke55 Yes, I know.
I meant that it's better to open an issue to track the problem.
Of course, I understand you won't forget to create a PR, but it's just in case.

Reply:
For example, if you create an issue like this, then we can close these conversations.

```
TODO: fix https://github.com/tier4/livox_ros2_driver/pull/10#discussion_r640260236 and https://github.com/tier4/livox_ros2_driver/pull/10#discussion_r640265140
```

Reply:
Will fix in https://github.com/tier4/livox_ros2_driver/issues/11